values/src/main/java/org/spine3/time/LocalDates.java

```diff
@@ -45,4 +46,22 @@ public static LocalDate today() {
                                           .build();
         return result;
     }
+
+    /**
+     * Obtains the date in the future after adding the specified amount of days.
```

Comment:
Do not describe what you do inside the method. Describe what we get as the result.